parity/cli/mod.rs

```diff
@@ -1027,23 +1021,55 @@ usage! {
 			"--gasprice=[WEI]",
 			"Equivalent to --min-gas-price WEI.",
 
-			ARG arg_etherbase: (Option<String>) = None, or |_| None,
-			"--etherbase=[ADDRESS]",
-			"Equivalent to --author ADDRESS.",
-
-			ARG arg_extradata: (Option<String>) = None, or |_| None,
-			"--extradata=[STRING]",
-			"Equivalent to --extra-data STRING.",
-
 			ARG arg_cache: (Option<u32>) = None, or |_| None,
 			"--cache=[MB]",
 			"Equivalent to --cache-size MB.",
 
-			ARG arg_tx_queue_ban_count: (u16) = 1u16, or |c: &Config| c.mining.as_ref()?.tx_queue_ban_count.clone(),
+			// ARG Removed in 1.7.
+
+			ARG arg_dapps_port: (Option<u16>) = None, or |c: &Config| c.dapps.as_ref()?.port.clone(),
+			"--dapps-port=[PORT]",
+			"Dapps server is merged with HTTP JSON-RPC server. Use --jsonrpc-port.",
```

Comment:
Should these arguments say instead that the dapps server has been removed?

Reply:
Yeah right. I checked, those arguments indeed does nothing.